presto-main/src/main/java/com/facebook/presto/operator/HttpPageBufferClient.java

```diff
@@ -275,9 +275,9 @@ private synchronized void initiateRequest()
         lastUpdate = DateTime.now();
     }
 
-    private void sendGetResults()
+    private synchronized void sendGetResults()
```

Comment:
Add a comment to the commit explaining why. This doesn't look like just fixing a warning.